src/EnergyPlus/DXCoils.cc

```diff
@@ -18407,10 +18315,8 @@ void CalcVRFCoilCapModFac(EnergyPlusData &state,
     }
 }
 
-Real64 FanSpdResidualCool([[maybe_unused]] EnergyPlusData &state,
-                          Real64 const FanSpdRto,          // indoor unit fan speed ratio
-                          std::array<Real64, 5> const &Par // parameters
-)
+Real64 FanSpdResidualCool(
```

Comment:
These two residual functions were exercised by unit tests, so I didn't want to get rid of them entirely.  The lambda above calls these functions still, but as you can see, these residual functions now accept a meaningful argument list.  There is no need to create an intermediate Par array, just capture the required variables in the lambda and pass them into this function call.  Then, again, inside this function, you don't need to get the data back off of the Par array.